slick.grid.js

```diff
@@ -4112,7 +4112,6 @@
       "getContentCanvasNode": getContentCanvasNode,
       "getTopCanvasNode": getTopCanvasNode,
       "focus": setFocus,
-      "updateAntiscroll": updateAntiscroll,

```

Comment:
This is technically a breaking change but since we don't use this anymore, I don't see why we need to keep it.